guides/source/configuring.md

```diff
@@ -707,6 +707,8 @@ There are a few configuration options available in Active Support:
 
 * `config.active_support.use_sha1_digests` specifies whether to use SHA-1 instead of MD5 to generate non-sensitive digests, such as the ETag header. Defaults to false.
 
+* `config.active_support.use_authenticated_message_encryption` specifies whether to use AES-256-GCM authenticated encryption as the default cipher for encrypting messages instead of AES-256-CBC. Defaults to true.
```

Comment:
The default is `false`.
https://github.com/rails/rails/blob/94360451eae2701fc04d6b132c44d9622bbe0cb5/activesupport/lib/active_support/message_encryptor.rb#L85
But this is enabled when defaults for Rails 5.2 is loaded. 
Therefore, I think that `This is false by default, but enabled when loading defaults for Rails 5.2.` is better.

Reply:
@y-yagi I think there is talk about default configurations, we probably should detect default value in
https://github.com/rails/rails/blob/master/railties/lib/rails/application/configuration.rb#L106. It is `true` there.

Reply:
The value is set only when `5.2`(or upper) is specified for `load_defaults`. 
It is set in the newly created application, but it is not always specified when upgrading from the old version.  
(I think that there are more applications in which it is not specified.)

Anyway, there is a possibility that it may not be set, so I think that the default should state the exact value.  
Otherwise, users using applications not specifying it will get confused.

Reply:
I think we may need to be more elaborate about this. The current wording _is_ confusing, as it uses `by default` right next to `defaults for Rails 5.2`, which _sound_ similar, to mean opposite things, without any additional explanation.